.github/workflows/lint.yml

```diff
@@ -51,6 +51,9 @@ jobs:
       - name: Build
         run: pipx run build
 
+      - name: Build
+        run: pipx run twine check --strict
```

Comment:
How is this different from the `pipx run build` above which creates the sdist and wheel to verify that building works?